CHANGELOG.md

```diff
@@ -3,8 +3,9 @@
 ## master / unreleased
 
 * [FEATURE] Ruler: Add `external_labels` option to tag all alerts with a given set of labels.
+* [FEATURE] Memberlist: Add `-memberlist.enable-broadcast-of-large-messages` option to enable broadcast of messages with more than 64KB.
```

Comment:
We should release this as part of 1.12, so can you kindly: 

* Make this PR's target to be the release-1.12 branch instead of master; We will merge from release branch to master later on. 
* Put this line under the 1.12.0 change log. 

Thank you so much for working on this feature flag,